### DISCUSSION TITLE
Video Processing Time

### DISCUSSION THREAD

**Discussion by mwilson703** [biigle/core#901] [Q&A]
Hi, I have been working on video annotations in a volume that was created on Thursday but a select few videos have not been processed. When I click on those videos I am met with the **'The video has not been processed yet. Please try again later'** message. I have tired putting these videos in a different volume but they are still unprocessed. I am wondering if there is a solution that I have not tired on my end or if more processing time is needed. Thank you!

**Comment by dlangenk** [comment]
Hi,

I am sorry for the late reply but the whole team was on holiday for the last two weeks. Usually the processing doesn't take that long, but if a huge number of videos/images/... is upload or processed it may take some time.
I hope the videos are processed now. If not can you please share the volume id or name or project id or name of the videos so I can have a look. If you do not want to disclose this information here, please write an email with a reference to this discussion and the info to info@biigle.de.

Best
Daniel
